frontend/app/forms/form-common-options/form-common-options.js

```diff
@@ -46,7 +48,38 @@ export const courtTypeOptions = [
   { label: "Justice Court", value: "Justice" }
 ];
 
-export function getJudicialDistrictFromCounty(value) {
+export function getJudicialDistrict(courtAddress, courtType) {
+  var judicialDistrict, county;
+  if (courtAddress) {
```

Comment:
When is `courtAddress` not being passed in? This function is returning `undefined` instead of an object when that happens, which seems a bit weird. Should we just ensure that courtAddress is always passed in, so we don't have to have this if statement?